GREEDY/LogHandler.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Linq;
+using System.Net.Http;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Web.Http;
+
+namespace GREEDY
+{
+    public class LogHandler : DelegatingHandler
+    {
+        private static object lockObject = new object();
+        private string logEntrySeperator => "---------------------------------------------------";
```

Comment:
change this to better looking 

private readonly logEntrySeperator = new String('-', n);, n-number or repetition

or using "string.Concat(Enumerable.Repeat("-", n));"

it would be better to change it to readonly